components/powersensor/ina/ina.go

```diff
@@ -252,14 +247,22 @@ func (d *ina) Voltage(ctx context.Context, extra map[string]interface{}) (float6
 }
 
 func (d *ina) Current(ctx context.Context, extra map[string]interface{}) (float64, bool, error) {
+
```

Comment:
Mild preference for removing this again, and not starting a new open block (function, for loop, if statement, etc.) with a blank line